src/qrb.c

```diff
@@ -42,3 +42,14 @@ int get_qrb(double *range, double *bearing) {
     return qrb(-1.0 * my.Long, my.Lat, -1.0 * DEST_Long, DEST_Lat,
 	       range, bearing);
 }
+
+bool get_qrb_for_locator(const char *locator, double *range, double *bearing) {
+    double dest_long, dest_lat;
+
+    if (RIG_OK != locator2longlat(&dest_long, &dest_lat, locator)) {
+	return false;   // invalid locator
+    }
+
```

Comment:
Comparing the use of 'qrb()' in 'get_qrb()' it seems we should have a look at our representation of the coordinates. The use of +/- prefix looks not intuitive.

Reply:
Unfortunately `cty.dat` and Hamlib have different orientation of longitudes. It is also commented in `qrb.c` (also in master).

Example: (-19.28 vs. 19.00)
```
 $ grep Hungary cty.dat
Hungary:                  15:  28:  EU:   47.12:   -19.28:    -1.0:  HA:
$ rotctl loc2lonlat jn97
19.000000
47.500000
```